applications/dashboard/models/class.usermodel.php

```diff
@@ -3454,75 +3454,113 @@ public function PasswordReset($UserID, $Password) {
     * @param array $User
     * @param boolean $PasswordOK
     */
-   public static function RateLimit($User, $PasswordOK) {
-      if (!Gdn::Cache()->ActiveEnabled()) return FALSE;
-//      $CoolingDown = FALSE;
+    public static function RateLimit($User, $PasswordOK) {
+        if (function_exists('apc_store')) {
```

Comment:
Ok, so I would rewrite it to be:

    if (Gdn::Cache()->ActiveEnabled())
        // Use cache
    elseif (C('Garden.Apc', false) && function_exists('apc_store'))
        // Use apc
    else
        // Use database (user attributes)

What do you think about  removing the large comment block of dead code?